src/python/build.sh

```diff
@@ -241,32 +257,32 @@ setup_and_run() {
             exit 1
         fi
 
-    elif [ "$option" == "TEST_DOC_BUILD" ]
+    elif [ "$OPTION" == "TEST_DOC_BUILD" ]
     then
-        echo "Running Sphinx doc build test"
+        echo "Running Sphinx docs test"
```

Comment:
we specifically should call this, doc(s) **build** test; as doctest are for different purpose